docker/jupyterlab/bashrc.felles

```diff
@@ -9,14 +9,6 @@ export FELLES=/ssb/bruker/felles
 # Adding pythonForSsb in PYTHONPATH
 export PYTHONPATH=$PYTHONPATH:/ssb/bruker/felles/pythonForSsb
 
-# Setting up environment variables for pip and pipenv
-# Pip config so users install from Nexus.
-export PIP_INDEX=http://pl-nexuspro-p.ssb.no:8081/repository/pypi-proxy/pypi
```

Comment:
Don't delete lines 14-17, we still need to use the nexus proxy, there's no opening in the firewall on-prem for PyPI access unfortunately

Reply:
Thanks @mmwinther, should be fixed now.